src/Adapters/AzureManagementAdapter.ts

```diff
@@ -157,8 +178,8 @@ export default class AzureManagementAdapter implements IAzureManagementAdapter {
      *  in their list of role assignments for a particular resource */
     async hasRoleDefinitions(
         resourceId: string,
-        uniqueObjectId: string,
-        accessRolesToCheck: AzureAccessPermissionRoleGroups
+        accessRolesToCheck: AzureAccessPermissionRoleGroups,
+        uniqueObjectId: string = this.uniqueObjectId
```

Comment:
uniqueObjectId is a class variable for AzureManagementAdapter that we pass in the constructor (it originally comes from auth when user logs in in 3rd party app, but in cardboard the value is from secrets file) so the default value in this method is the adapters its own variable, otherwise for different users this adapter method still can check **role assignments** for different users if the consumer of this method wants to.

Reply:
gotcha. it's the User's object id then. Makes sense.